consensus/hotstuff/pacemaker/timeout/config_test.go

```diff
@@ -9,41 +9,34 @@ import (
 )
 
 func TestConstructor(t *testing.T) {
-	c, err := NewConfig(2200*time.Millisecond, 1200*time.Millisecond, 0.73, 1.5, 0.85, time.Second)
+	c, err := NewConfig(2200*time.Millisecond, 1200*time.Millisecond, 1.5, 0.85, time.Second)
 	require.NoError(t, err)
 	require.Equal(t, float64(2200), c.ReplicaTimeout)
 	require.Equal(t, float64(1200), c.MinReplicaTimeout)
-	require.Equal(t, float64(0.73), c.VoteAggregationTimeoutFraction)
 	require.Equal(t, float64(1.5), c.TimeoutIncrease)
 	require.Equal(t, float64(0.85), c.TimeoutDecrease)
 	require.Equal(t, float64(1000), c.BlockRateDelayMS)
 
 	// should not allow startReplicaTimeout < minReplicaTimeout
-	_, err = NewConfig(800*time.Millisecond, 1200*time.Millisecond, 0.73, 1.5, 0.85, time.Second)
+	_, err = NewConfig(800*time.Millisecond, 1200*time.Millisecond, 1.5, 0.85, time.Second)
 	require.Error(t, err)
```

Comment:
could we please replace _all_ `require.Error(t, err)` in this test by 
```golang
require.True(t, model.IsConfigurationError(err))
```
thanks